gramps_webapi/translogger.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2005 Ian Bicking and contributors; written for Paste (http://pythonpaste.org)
+# Licensed under the MIT license: http://www.opensource.org/licenses/mit-license.php
+"""
+Middleware for logging requests, using Apache combined log format
+"""
+
+import logging
+import time
+import urllib.parse
+
+
+class TransLogger(object):
```

Comment:
`(object)` is not needed (I know this is copy-pasted from somewhere else, but we can modernize a bit)

Reply:
I can fix a couple of other things there too. Now that I know you are open to this PR, I will do that.

Reply:
Updated!